src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -311,7 +313,7 @@ export const SearchBarDropdown = ({
                 isLoading={!trendingTokenData}
               />
             )}
-            {!isTokenPage && (
+            {!isTokenPage && !shouldDisableNFTRoutes && (
```

Comment:
nit: try to avoid compound boolean statements as render conditions

Reply:
this is fine as-is, because both are booleans, but there is an edge case where you may accidentally render a non-boolean falsey value. To be safe, you could make it a simple boolean:

```suggestion
            {Boolean(!isTokenPage && !shouldDisableNFTRoutes) && (
```